ecommerce_extensions/payment/encryption_utils.py

```diff
@@ -0,0 +1,39 @@
+""" Moduel to encript the order number in the url for status page """
+import base64
+
+from cryptography.fernet import Fernet
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.kdf.pbkdf2 import PBKDF2HMAC
+from django.conf import settings
+
+
+def generate_key():
+    """ Generate the key for encription
+    """
+    encryption_key = getattr(settings, 'PAYMENT_ENCRYPTION_KEY', 'encryption_key')
+    encryption_key = encryption_key.encode()  # Convert to type bytes
+    kdf = PBKDF2HMAC(
```

Comment:
what is kdf?

Reply:
@andrey-canon

Reply:
kdf is short for Key Derivation Function, commonly used named in cryptography.

PB - KDF - 2HMAC